mwdb/web/src/commons/config/provider.js

```diff
@@ -88,6 +88,12 @@ export function ConfigProvider(props) {
         if (auth.isAuthenticated) updateRemoteInfo();
     }, [auth.isAuthenticated]);
 
+    useEffect(() => {
+        if (Number.isInteger(serverConfig.config["request_timeout"])) {
```

Comment:
What if it's set to zero? I think we should just check for truthy values.
```suggestion
        if (serverConfig.config["request_timeout"]) {
```

Reply:
Ok, I'm revoking that comment: request_timeout is initially set for `/api/server`

Reply:
Explained during meeting with @psrok1 . Initial value for request_timeout is establish during creation Axios object. After retrieving value from server (mwdb.ini file) it is changed to this value.